server/form-pages/apply/basic-information/placementDate.test.ts

```diff
@@ -0,0 +1,64 @@
+import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../shared-examples'
+
+import PlacementDate from './placementDate'
+
+describe('PlacementDate', () => {
+  itShouldHaveNextValue(new PlacementDate({}), '')
+  itShouldHavePreviousValue(new PlacementDate({}), 'oral-hearing')
+
+  describe('errors', () => {
+    it('should return an empty array if the release date is the same as the start date', () => {
+      const page = new PlacementDate({
+        startDateSameAsReleaseDate: 'yes',
+      })
+      expect(page.errors()).toEqual([])
+    })
+
+    it('should return an empty array if the release date is the same as the start date and the date is specified', () => {
```

Comment:
Should this be '...if the release date is the same as the start date'. I don't think some of the other names below are accurate either. 
It might be easier to parse if we have two seperate `describe` blocks - one for when the release date is the same as the start date and one for when it isn't

Reply:
Good plan. I've fixed it for all the other date-shaped pages too. Much nicer!